types/marked/index.d.ts

```diff
@@ -95,7 +95,7 @@ declare namespace marked {
         html(html: string): string;
         heading(text: string, level: number, raw: string, slugger: Slugger): string;
         hr(): string;
-        list(body: string, ordered: boolean, start: number): string;
+        list(body: string, ordered: boolean, start: number | ''): string;
```

Comment:
I'm not sure that we'd want to do this? I see that `startatt` (https://github.com/markedjs/marked/blob/master/lib/marked.js#L990) may possibly be assigned the eventual value of `''`, but it doesn't seem like this interface is meant to take the empty string.

Reply:
Unfortunately, There's a possibility that the `start` type is string.
https://github.com/markedjs/marked/blob/master/lib/marked.js#L327

For stack:
1. https://github.com/markedjs/marked/blob/master/lib/marked.js#L1259
2. https://github.com/markedjs/marked/blob/master/lib/marked.js#L1253
3. https://github.com/markedjs/marked/blob/master/lib/marked.js#L1158
4 https://github.com/markedjs/marked/blob/master/lib/marked.js#L331